backend/src/main/java/com/rcbsukcesja/hack2react/service/OrganizationNGOService.java

```diff
@@ -202,6 +203,12 @@ public OrganizationNGOView patchUpdateNGO(UUID ngoId, OrganizationNGOPatchDto dt
         if (dto.legalStatus() != null && !actual.getLegalStatus().equals(dto.legalStatus())) {
             actual.setLegalStatus(dto.legalStatus());
         }
+        if(dto.disabled() != actual.isDisabled()){
+            actual.setDisabled(dto.disabled());
+        }
+        if(dto.reason() != null && !dto.reason().equals(actual.getReason())){
+            actual.setReason(dto.reason());
+        }
```

Comment:
Musimy sprawdzać, czy jak ktoś daje disabled na true, to czy dał powód zablokowania (to samo trzeba dać do puta pod confirmed):
```suggestion
        if (dto.disabled() != null) {
            if (dto.disabled()) {
                 if (dto.reason() == null) {
                       // tutaj exception "Reason must not be null when disabling NGO"
                 }
            }
            ngo.setDisabled(dto.disabled());
        } 
        if(dto.reason() != null && !dto.reason().equals(actual.getReason())){
            actual.setReason(dto.reason());
        }
```